srcpkgs/opencl-amd/template

```diff
@@ -0,0 +1,65 @@
+# Template file for 'opencl-amd'
+pkgname=opencl-amd
+version=18.50.708488
+revision=1
+create_wrksrc=yes
+hostmakedepends="wget"
+depends="ocl-icd"
+short_desc="OpenCL userspace driver as provided by the amdgpu-pro driver stack"
+maintainer="Adam Buechner <adi51@gmx.de>"
+license="custom"
+homepage="http://www.amd.com"
+archs="x86_64"
+repository=nonfree
+
+_prefix="amdgpu-pro-"
+_postfix="-ubuntu-18.04"
+_major="18.50"
```

Comment:
Please do it like here: https://github.com/void-linux/void-packages/blob/master/srcpkgs/ebtables/template#L5 to keep version defined once.